queuebacca-sqs/src/main/java/io/axonif/queuebacca/sqs/SqsCourierMessageBinRegistry.java

```diff
@@ -200,11 +214,25 @@ private void setQueueAttributes(String queueUrl, Duration visibilityTimeout, Con
         client.setQueueAttributes(attributesRequest);
     }
 
-    private void setTags(String queueUrl, Collection<SqsTag> tags) {
+    private void setTags(String queueUrl, Collection<SqsTag> tags, Configuration configuration) {
         Map<String, String> tagMap = tags.stream().collect(Collectors.toMap(SqsTag::getKey, SqsTag::getValue));
         TagQueueRequest tagQueueRequest = new TagQueueRequest(queueUrl, tagMap);
 
-        client.tagQueue(tagQueueRequest);
+        for(int i = 0; i < configuration.getInt(SQS_TAGGING_RETRY_ATTEMPTS_PROPERTY, SQS_TAGGING_RETRY_ATTEMPTS_DEFAULT); i++) {
```

Comment:
So now if we fail to tag because of being throttled 5 times, we just don't tag right?  Maybe we should log a warning just so people know what has happened?

Reply:
Yea true, I should probably log it as a warning. I may rejig this code to make it cleaner with that requirement added.